packages/gatsby-tinacms-mdx/src/mdxFormHoc.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+import { FormOptions, Form } from 'tinacms'
+import { TinaForm } from '@tinacms/form-builder'
```

Comment:
`TinaForm` can be imported directly from `tinacms`. This will solve @paulbunker's issue with needing `form-builder` as a dep